### PR TITLE
feat: Upgrade `actions/core` and `octokit` deps

### PR DESCRIPTION
**Pull request by billyvg**

### CODE REVIEW

**Review by HTC37 — COMMENTED** [review]
gh pr checkout 16

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
         yarn format-check
 
     - name: build
-      run: yarn build
+      run: yarn dist
```
Comment (shenotme) [inline .github/workflows/test.yml:23]: End

Comment (shenotme) [inline .github/workflows/test.yml:23]: End

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ jobs:
         yarn format-check
 
     - name: build
```
Comment (shenotme) [inline .github/workflows/test.yml:22]: End

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -20,7 +20,7 @@ jobs:
...
-      run: yarn build
```
Comment (shenotme) [inline .github/workflows/test.yml:23]: End

Comment (shenotme) [inline .github/workflows/test.yml:23]: End

File: package.json [hunk 4]
```diff
@@ -45,27 +45,27 @@
     "node-fetch": "^2.6.1"
   },
   "dependencies": {
-    "@actions/core": "^1.2.6",
-    "@octokit/auth-app": "^2.4.14",
-    "@octokit/rest": "^18.0.3"
```
Comment (shenotme) [inline package.json:50]: End

File: package.json [hunk 5]
```diff
@@ -45,27 +45,27 @@
     "node-fetch": "^2.6.1"
   },
   "dependencies": {
```
Comment (shenotme) [inline package.json:47]: End

File: package.json [hunk 6]
```diff
@@ -45,27 +45,27 @@
...
-    "@actions/core": "^1.2.6",
-    "@octokit/auth-app": "^2.4.14",
```
Comment (shenotme) [inline package.json:49]: End

File: package.json [hunk 7]
```diff
@@ -45,27 +45,27 @@
     "node-fetch": "^2.6.1"
   },
   "dependencies": {
-    "@actions/core": "^1.2.6",
```
Comment (shenotme) [inline package.json:48]: End

File: src/main.ts [hunk 8]
```diff
@@ -47,7 +49,9 @@ async function run(): Promise<void> {
     // @ts-expect-error
     core.setOutput('token', resp.token);
   } catch (error) {
-    core.setFailed(error.message);
```
Comment (shenotme) [inline src/main.ts:50]: End

File: src/main.ts [hunk 9]
```diff
@@ -47,7 +49,9 @@ async function run(): Promise<void> {
...
+    if (error instanceof Error) {
+      core.setFailed(error.message);
```
Comment (shenotme) [inline src/main.ts:53]: End

File: src/main.ts [hunk 10]
```diff
@@ -47,7 +49,9 @@ async function run(): Promise<void> {
...
+    }
```
Comment (shenotme) [inline src/main.ts:54]: Karma Mac book air

**Pull request merged**